manifests/install.pp

```diff
@@ -53,13 +57,13 @@
       revision => $version,
     }
   } else {
-    package { 'letsencrypt':
+    package { $package_name:
```

Comment:
> In the adding relationship case, wouldn't it be expected to have working Package[$my_pkg] when providing $package_name, eg. class {'letsencrypt': package_name => $my_pkg}? After all it is named "package_name".

Yeah, if you're passing in package_name then I guess you know it - but if it's derived from params.pp then you might not.

It's probably personal preference.